### PR TITLE
docs: adding the package catalog docs

### PR DESCRIPTION
**Pull request by leoporoli**
## Description:
The package catalog logs

## Is this change user facing?
YES

### CODE REVIEW

File: docs/docs/get-started/package-catalog.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+title: Package catalog
+sidebar_label: Basic Concepts
+slug: '/package-catalog'
+sidebar_position: 6
+---
+
+Package catalog
+---------------
+
+The Kurtosis package catalog if the public repository of Kurtosis packages where package authors can publish their packages which later can be found and used by any user.
```
Comment (leeederek) [inline docs/docs/get-started/package-catalog.md]:
```suggestion
The Kurtosis package catalog is a public repository of Kurtosis packages where package authors can publish their packages which later can be found and used by any user.
```

File: docs/docs/get-started/package-catalog.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+title: Package catalog
```
Comment (leeederek) [inline docs/docs/get-started/package-catalog.md]:
```suggestion
title: Package Catalog
```

File: docs/docs/get-started/package-catalog.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+slug: '/package-catalog'
+sidebar_position: 6
+---
+
+Package catalog
+---------------
+
+The Kurtosis package catalog if the public repository of Kurtosis packages where package authors can publish their packages which later can be found and used by any user.
+
+The catalog address is [catalog.kurtosis.com][package-catalog]
```
Comment (leeederek) [inline docs/docs/get-started/package-catalog.md]:
```suggestion
The Package Catalog can be found at: [catalog.kurtosis.com][package-catalog]
```

File: docs/docs/get-started/package-catalog.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+
+How can authors add a new package into the catalog?
```
Comment (leeederek) [inline docs/docs/get-started/package-catalog.md]:
```suggestion
How can do I add my package to the catalog?
```

File: docs/docs/get-started/package-catalog.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+---------------------------------------------------
+
+The catalog is made up of a curated list that is stored in the `kurtosis-package-catalog.yml` file within the [kurtosis-package-catalog repository.][package-catalog-repository]
+
+If an author wants to add their package to the catalog, they should add its name on the list inside the `kurtosis-package-catalog.yml` file and send a pull request to be validated by the CI validations and approved by a Kurtosis administrator.
```
Comment (leeederek) [inline docs/docs/get-started/package-catalog.md]:
```suggestion
If an author wants to add their package to the catalog, they should add its name on the list inside the `kurtosis-package-catalog.yml` file and send a pull request to be validated by the CI validations and approved by a Kurtosis administrator. While we do not have any SLAs, the Kurtosis team will periodically review pull requests. Should you need any help or have questions, we're always available in our [Discord server](https://discord.gg/twRRs8GxMD).
```

File: docs/docs/get-started/package-catalog.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+### What validates the CI?
+
+The CI's automatic jobs validate the following for each new Pull Request created:
+
+- That there are no duplicated names within the `kurtosis-package-catalog.yml` file, there cannot be two or more packages with the same name.
+- That the package repository exists.
```
Comment (leeederek) [inline docs/docs/get-started/package-catalog.md]: Question: does it need to exist *publicly* too?

File: docs/docs/get-started/package-catalog.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+- That the repository contains the `kurtosis.yml` file.
+- That the name declared in the `kurtosis.yml` file corresponds to the name that is being added to the catalog
+- Only if the package contains an icon (you should upload an image file named `kurtosis-package-icon.png` in your package repository, at the same level as the `kurtosis.yml` file, is if you want to add an icon to be displayed in the catalog)
```
Comment (leeederek) [inline docs/docs/get-started/package-catalog.md]:
```suggestion
- If the package contains an icon (you should upload an image file named `kurtosis-package-icon.png` in your package repository, at the same level as the `kurtosis.yml` file, is if you want to add an icon to be displayed in the catalog), then:
```

**Review by leeederek — APPROVED** [review]

File: docs/docs/get-started/package-catalog.md (hunk 6 above)
Comment (leoporoli) [reply docs/docs/get-started/package-catalog.md]: yes, it has to be public @leeederek

File: docs/docs/get-started/package-catalog.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+The catalog is made up of a curated list that is stored in the `kurtosis-package-catalog.yml` file within the [kurtosis-package-catalog repository.][package-catalog-repository]
+
+If an author wants to add their package to the catalog, they should add its name on the list inside the `kurtosis-package-catalog.yml` file and send a pull request to be validated by the CI validations and approved by a Kurtosis administrator. While we do not have any SLAs, the Kurtosis team will periodically review pull requests. Should you need any help or have questions, we're always available in our [Discord server](https://discord.gg/twRRs8GxMD).
+
+### What validates the CI?
+
+The CI's automatic jobs validate the following for each new Pull Request created:
+
+- That there are no duplicated names within the `kurtosis-package-catalog.yml` file, there cannot be two or more packages with the same name.
+- That the package repository exists.
```
Comment (leoporoli) [inline docs/docs/get-started/package-catalog.md]:
```suggestion
- That the package repository exists and it's a public repository.
```

File: docs/docs/get-started/package-catalog.md (hunk 6 above)
Comment (leoporoli) [reply docs/docs/get-started/package-catalog.md]: I made the change

**Pull request merged**